test/integration/targets/azure_rm_virtualmachine/tasks/virtualmachine.yml

```diff
@@ -101,20 +92,20 @@
       that:
         - azure_vm.properties.availabilitySet.id
 
-- add_host:
-      name: new_azure_vm
-      ansible_host: '{{ output.ansible_facts.azure_vm.properties.networkProfile.networkInterfaces[0].properties.ipConfigurations[0].properties.publicIPAddress.properties.ipAddress }}'
-      ansible_connection: paramiko  # not guaranteed to have sshpass...
-      ansible_user: adminuser
-      ansible_password: Password123!
-      ansible_host_key_checking: false
-
-- name: wait for SSH port to be open
-  wait_for:
-      host: '{{ hostvars["new_azure_vm"].ansible_host }}'
-      port: 22
-      timeout: 60
-      state: started
```

Comment:
why commented out?

Reply:
they're used for below commented out test, after 22 port  ready, install something on the vm, but below tests are commented  out already, so this waifor is meaningless.

Reply:
yes, seems like the commented out test is very unstable.... so just wanted to confirm if you found similar issue :-)